quickstarts/atlassian/atlassian-jira/config.yml

```
@@ -0,0 +1,47 @@
+# Name of the quickstart (required)
+name: atlassian-jira
+
+# Displayed in the UI (required)
+title: Atlassian JIRA
+
+# Long-form description of the quickstart (required)
+description: |
+  Jira is a proprietary issue tracking product developed by Atlassian that allows bug tracking and agile project management. You can Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
+
+  Review this integration guide for more [information](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#jira).
+
+
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+summary: |
+  Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
+
+# Support level: New Relic | Verified | Community (required)
+level: Verified
+
+# Authors of the quickstart (required)
+authors:
+  - Matan Moser (New Relic)
+  - Ismail Azam (New Relic)
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - JIRA
+  - Atlassian
+  - Notifications
+  - AIOps
+  - Incident Intelligence
+  - Destinations
+
+installPlans:
+  - third-party-atlassian-jira
```

The path is not factored into the id of an install plan, you'll just need to update this:
```suggestion
  - atlassian-jira
```

---

see my recent commit which sets the install plan to third-party-atlassian-jira, which follows the naming convention for all install.ym files in the third party directory https://github.com/newrelic/newrelic-quickstarts/tree/main/install/third-party

---

Notice that those install plan ids have the string `third-party` prefixed to their ids. The install plan for this quickstart does not have that prefix.

---

the sendgrid third party follows the pattern I chose to go with where the installPlan ID is  `third-party-atlassian-jira` and the reference to the plan is  `third-party-atlassian-jira`

I didn't check everyone though, but I assume this is the pattern to follow and as long as the `id` matches the the reference to that `plan` it will work.

as a note: this is why I want to document the creation of the install plans better.

---

it looks like i didn't push... 💯